src/Theme/index.js

```diff
@@ -85,6 +95,33 @@ const TextWrapper = styled(Text)`
   color: ${({ color, theme }) => theme[color]};
 `;
 
+export const Typography = {
+  custom: ({ color, ...rest }) => <TextWrapper color={color || 'text1'} {...rest} />,
+
+  tinyText: ({ sx, color, ...rest }) => (
+    <TextWrapper fontWeight={400} fontSize={'8px'} lineHeight={'10px'} color={color || 'text1'} sx={sx} {...rest} />
+  ),
+  smallText: ({ sx, color, ...rest }) => (
+    <TextWrapper fontWeight={400} fontSize={10} lineHeight={'10px'} color={color || 'text1'} sx={sx} {...rest} />
+  ),
+  text: ({ sx, color, ...rest }) => (
+    <TextWrapper fontWeight={400} fontSize={13} lineHeight={'16px'} color={color || 'text1'} sx={sx} {...rest} />
+  ),
+  largeText: ({ sx, color, ...rest }) => (
+    <TextWrapper fontWeight={400} fontSize={14} lineHeight={'17px'} color={color || 'text1'} sx={sx} {...rest} />
+  ),
+
+  smallHeader: ({ sx, color, ...rest }) => (
+    <TextWrapper fontWeight={400} fontSize={16} lineHeight={'19px'} color={color || 'text1'} sx={sx} {...rest} />
+  ),
+  largeHeader: ({ sx, color, ...rest }) => (
+    <TextWrapper fontWeight={400} fontSize={24} lineHeight={'29px'} color={color || 'text1'} sx={sx} {...rest} />
+  ),
+  largeBoldHeader: ({ sx, color, ...rest }) => (
+    <TextWrapper fontWeight={700} fontSize={24} lineHeight={'29px'} color={color || 'text1'} sx={sx} {...rest} />
+  ),
```

Comment:
Maybe remove the rest spreading. I dont think we need that. Just sx prop is enough for styling